components/common/ErrorRuntime.tsx

```diff
@@ -0,0 +1,17 @@
+import MainLayout from '../layout/MainLayout';
+import ResultsRetryButton from '../results/ResultsRetryButton';
+
+import CenteredCard from './CenteredCard';
+import CenteredCardContent from './CenteredCardContent';
+
+export default function ErrorRuntime(): JSX.Element {
+  return (
+    <MainLayout>
+      <CenteredCard>
+        <CenteredCardContent title="An error occured" description="Sorry, an error occurred. Please try again." iconClasses="fa fa-exclamation-circle text-warning mb-4">
```

Comment:
occured should have 2 `r`s like "occurred". I missed this in the title on the first review.

Reply:
Good catch, resolved!